ChainRouter.php

```diff
@@ -248,10 +248,10 @@ public function generate($name, $parameters = array(), $absolute = false)
         throw new RouteNotFoundException(sprintf('None of the chained routers were able to generate route: %s', $info));
     }
 
-    private function getErrorMessage($name, $router = null, $parameters = null)
+    private function getErrorMessage($name, $router = null, $parameters = array())
     {
         if ($router instanceof VersatileGeneratorInterface) {
-            $displayName = $router->getRouteDebugMessage($name, $parameters);
+            $displayName = $router->getRouteDebugMessage($name, $parameters ?: array());
```

Comment:
no need for this change, $parameters will now always be an array. and this method is private, so there are no BC considerations.